specs/swagger.yaml

```diff
@@ -198,7 +197,95 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-
+  /psms:
+     get:
+      summary: Get a collection of peptide spectrum matches  
+      operationId: getPSMs
+      description: Get specific Peptide Spectrum Match (PSM) if is present in the database and has been identified by a previous experiment in the resource. 
```

Comment:
Pluralize all references to PSMs in this sentence , since the endpoint retrieves a collection. i.e. "Get specific Peptide Spectrum Match**es** (PSM**s**) if **they are** present in the database and **have** been identified by a previous experiment in the resource."